src/lightning/app/utilities/packaging/cloud_compute.py

```diff
@@ -81,6 +81,9 @@ class CloudCompute:
 
         mounts: External data sources which should be mounted into a work as a filesystem at runtime.
 
+        colocation_group_id: Identifier for groups of works to be colocated in the same datacenter.
+            Can be a string of max. 64 characters.
```

Comment:
```suggestion
            Set this to a string of max. 64 characters and all works with this group id will run in the same datacenter.
            If not set, the works are not guaranteed to be colocated.
```
@nicolai86 Just a suggestion to make it a bit more explicit that the user chooses a name. I'm not sure about the last sentence, it's just what I guessed would be the default behavior.